lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.scss

```diff
@@ -92,3 +85,12 @@
         box-shadow: 0 8px 9px -5px var(--adf-theme-foreground-divider-color), 0 15px 22px 2px var(--adf-theme-foreground-divider-color);
     }
 }
+
+.adf-datatable .adf-datatable-row .adf-datatable-cell-value .adf-permission-role-column-header {
```

Comment:
Why are you overriding these styles here instead of in component where adf-permission-role-column-header class is used? This class is not in html of permission-list component. You can override styles for adf-permission-role-column-header in component which has that class in html.